ibis/pandas/udf.py

```diff
@@ -260,11 +284,9 @@ def wrapper(func):
         # Don't reorder the multiple dispatch graph for each of these
         # definitions
         with pause_ordering():
```

Comment:
I didn't make this API, but if I had to guess, it's because it prevents reordering of the type signature graph in calls to `.register`. It's cheaper to amortize the cost of the ordering over as many definitions as you can fit into a call to `pause_ordering` especially if you have a lot of different versions of a function (which we do with `execute_node`).

This isn't necessary after `multipledispatch=0.6.0` because the type signature graph of a function will be ordered when it is first called so it's effectively always paused until you call the function.